plugins/metadata/dc11/ru_RU/locale.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale.xml
```

Comment:
Thanks, @ppv1979 and @mtub -- I'll make a note to fix the en_US locales sometime. Obviously not a big deal, and thanks for keeping everything consistent.